packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -191,7 +191,11 @@ contract EpochManager is
    */
   function startNextEpochProcess() external nonReentrant onlySystemAlreadyInitialized {
     require(isTimeForNextEpoch(), "Epoch is not ready to start");
-    require(!isOnEpochProcess(), "Epoch process is already started");
+    require(
+      epochProcessing.status != EpochProcessStatus.Started,
```

Comment:
if we would change this to require(epochProcessing.status == EpochProcessStatus. NotStarted) it would solve current issue and also if we would add new state in the future this wouldn't have to change either